<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectDockWidget</class>
 <widget class="QDockWidget" name="KinectDockWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string notr="true">Kinect</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="startScanPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Scan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="vrtRadioButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>158</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>watch real time</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="vsmRadioButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>158</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>view scanned model</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="saveScanPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Scan</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="tiltSpinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
    <property name="maximum">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Tilt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCalibrationPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load a YML calibration file as written by Nicholas Burrus kinect calibration software: http://nicolas.burrus.name/index.php/Research/KinectCalibration</string>
    </property>
    <property name="whatsThis">
     <string>Load a YML calibration file as written by Nicholas Burrus kinect calibration software: http://nicolas.burrus.name/index.php/Research/KinectCalibration</string>
    </property>
    <property name="text">
     <string>Load Calibration</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
